lms/config.py

```diff
@@ -105,6 +105,7 @@ def __post_init__(self):
     _Setting("mailchimp_digests_subaccount"),
     _Setting("mailchimp_digests_email"),
     _Setting("mailchimp_digests_name"),
+    _Setting("email_preferences_secret"),
```

Comment:
We'd need the `passenv` in tox and devdata value for this one

Reply:
Added

Reply:
Good catch on the devdata PR. I went ahead and took the liberty of merging this without review: https://github.com/hypothesis/devdata/pull/101